src_main/xevem_type.h

```diff
@@ -36,7 +36,7 @@
 #ifndef _XEVEM_TYPE_H_
 #define _XEVEM_TYPE_H_
 
-#include <xeve_type.h>
+#include "xeve_type.h"
```

Comment:
@Jamaika1 Seems this is not related to the main goal of your PR at this moment. For a better `git log` historical management for our community, what about separating this commit as a separate PR?